tests/drivers/dma/chan_blen_transfer/src/test_dma.c

```diff
@@ -36,7 +36,7 @@ static char rx_data[RX_BUFF_SIZE] = { 0 };
 static void test_done(const struct device *dma_dev, void *arg,
 		      uint32_t id, int error_code)
 {
-	if (error_code == 0) {
+	if (error_code >= 0) {
```

Comment:
no longer an error code, should be replaced with status

Reply:
I also changed the message printed to print out the status